websockets.md

```diff
@@ -205,6 +206,8 @@ websocket.onerror = function(evt) { onError(evt) };
 ```
 {: codeblock}
 
+The client can open multiple concurrent WebSocket connections to the service. The number of concurrent connections is limited only by the capacity of the service, which should pose no problems for most users.
```

Comment:
"which generally do not pose problems for users"? (conditional!)

Reply:
AAAHHHH!  Next thing you know....